index.js

```diff
@@ -30,8 +31,12 @@ exports.parse = function parseIntelHex(data, bufferSize) {
 		highAddress = 0, //upper address
 		startSegmentAddress = null,
 		startLinearAddress = null,
+		addressOffset = 0 || addressOffset,
```

Comment:
Should be `addressOffset ||= 0`

Reply:
That's a good point (although still rarely seen) but I have to use Node 12 and from what I understand this would bump the requirements to Node 15